src/main/java/com/emc/mongoose/base/load/step/local/context/LoadStepContextImpl.java

```diff
@@ -303,10 +303,14 @@ public final boolean put(final O opResult) {
 				counterResults.increment();
 			}
 		} else if (Status.PENDING.equals(status)) {
-			//in case driver cannot finish operation due to storage API issues or of some other sort, we need
-			//to set the operation status to Pending, so that we don't count it in the metrics and recycle the operation
+			// in case driver cannot finish operation due to storage API issues or of some other sort, we need
+			// to set the operation status to Pending, so that we don't count it in the metrics and recycle the operation
 			generator.recycle(opResult);
 			counterResults.increment();
+		} else if (Status.OMIT.equals(status)) {
+			// operation status is set to Pending in case we want an operation to complete, but not to register
```

Comment:
I assume there's a typo in the comment: `Pending` -> `Omit`

Reply:
it was a typo, yes, it's fixed now